wordpress-mysql-replication/mainTemplate.json

```diff
@@ -0,0 +1,384 @@
+{
```

Comment:
Will azuredeploy be going to marketplace as well? I suggest having just azuredeploy while we sort out the direction on integrating marketplace templates here